include/psa/crypto_values.h

```diff
@@ -328,6 +329,13 @@
  */
 #define PSA_ERROR_DATA_INVALID          ((psa_status_t)-153)
 
+/** The function that returns this error is defined as interruptible and
+ *  still has work to do, thus the user should call the function again with the
+ *  same parameters until it either returns #PSA_SUCCESS or any other error.
+ *  This is not an error per se, more a notification of status.
+ */
+#define PSA_ERROR_IN_PROGRESS           ((psa_status_t)-154)
```

Comment:
Earlier discussion on this status code resulted in an initial agreement that `PSA_OPERATION_IN_PROGRESS` would be a good name. And we picked a value at the end of the reserved values &mdash; `-248` &mdash; to separate it from the error codes:

* 'Error' in the name suggests that something is wrong, and that progress is not possible.
* A negative result code is used to indicate that the operation has not succeeded, and to allow APIs that use this status code to return positive, non-zero success codes.

I like some of the wording you have used for the description of this status code, and may use that in the common definition.

See https://github.com/ARM-software/psa-ff-m/pull/322#discussion_r936521079